packages/react/src/components/Tag/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Tag, { TagLabel } from '../../../src/components/Tag';
+import axe from '../../axe';
+
+test('renders children', () => {
+  render(
+    <Tag>
+      <TagLabel>Label:</TagLabel> value
+    </Tag>
+  );
+  expect(screen.getByText('Label:')).toBeInTheDocument();
+});
```

Comment:
This test seems redundant because the ones below already verify that it renders.